step-04/index.js

```diff
@@ -29,15 +29,15 @@ io.sockets.on('connection', function(socket) {
   socket.on('create or join', function(room) {
     log('Received request to create or join room ' + room);
 
-    var numClients = io.sockets.sockets.length;
+    var clientsInRoom = io.sockets.adapter.rooms[room];
```

Comment:
This doesn't seem to work for me. The room name `room` is from the prompt ('foo' or whatever) whereas `io.sockets.adapter.rooms` returns objects keyed by their unique IDs, not room names, so `clientsInRoom` is always zero.

I like @AlexRobinson's approach here: https://github.com/googlecodelabs/webrtc-web/issues/5.

If you're OK with that, could you update this and the other index.js files?

Thanks again!